Terminal.Gui/Views/ScrollView.cs

```diff
@@ -38,29 +38,32 @@ public ContentView (Rect frame) : base (frame)
 
 		ContentView contentView;
 		ScrollBarView vertical, horizontal;
+		int scrollLines, scrollCols;
 
 		/// <summary>
 		///  Initializes a new instance of the <see cref="Gui.ScrollView"/> class using <see cref="LayoutStyle.Absolute"/> positioning.
 		/// </summary>
 		/// <param name="frame"></param>
-		public ScrollView (Rect frame) : base (frame)
+		/// <param name="scrollLines">Scroll amount for vertical scrolling</param>
```

Comment:
`scrollLines` should be `scrollRows`.

Is there a known use-case where `scrollLines` is different than `scrollCols`?  Just want to make sure we're not adding complexity if it's not needed.

Reply:
I don't think so, it's just that in `public ScrollBarView (View host, bool isVertical, bool showBothScrollIndicator = true) : this (0, 0, isVertical)` I saw that we're initializing another scroll indicator if `showBothScrollIndicator`, so I wanted to work on giving developers a way to initialize it independently of the first one. Though you're right, it does add a bit of complexity without offering a significant value, so I will probably reduce these two variables to just one `scrollAmount` or similar.

Reply:
One situation that can be justified to have different values for lines and cols is if the size of them are very different. So, on a smaller size we may scroll less than a bigger one. In this case I can propose a percentage amount. Thus the scrolling is proportional to his size. Well only an idea.